vitals/BatteryFactorValidator.java

```diff
@@ -0,0 +1,25 @@
+package vitals;
+public class BatteryFactorValidator {
+  
+	public static boolean EvaluateBatteryMaxMeasure(String ParamName,float ParamValue,float MaxParamLimit) {
+		
+        return ParamValue>MaxParamLimit;
+	}
+	
+	public static boolean EvaluateBatteryMinMeasure(String ParamName,float ParamValue,float MinParamLimit) {
+		return ParamValue < MinParamLimit;
+	}
+	
+	
+    public static boolean CheckLBreach(String ParamName,float ParamValue,float MinParamLimit,float MaxParamLimit)

```

Comment:
what does CheckL and CheckH stands for ?
function name should be in camel case

Reply:
changed method names to checkLowBreach and checkHighBreach functions